src/index.js

```diff
@@ -45,6 +45,10 @@ Chart.registry = registry;
 Chart.Scale = Scale;
 Chart.Ticks = Ticks;
 
+// Compatibility with ESM extensions
+Object.assign(Chart, controllers, scales, elements, plugins, platforms);
```

Comment:
not sure whether that works in minified cases

Reply:
Works fine, exports / properties are not mangled